src/OpenLoco/Widget.cpp

```diff
@@ -11,40 +11,36 @@ using namespace OpenLoco::Interop;
 
 namespace OpenLoco::Ui
```

Comment:
You've changed this to repeat `Widget::` in function signatures. Why not just do:
```suggestion
namespace OpenLoco::Ui::Widget
```